content/threads_intro.tex

```diff
@@ -52,9 +52,8 @@
 In addition, a blocked thread will not prevent the progress of \openshmem calls
 performed on other \acp{PE}.
 
-\item In the \CONST{SHMEM\_THREAD\_MULTIPLE} thread level, all \openshmem calls are thread-safe.
-Any two concurrently running threads may make \openshmem calls and the outcome
-will be as if the calls executed in some order, even if their execution is interleaved.
+\item In the \CONST{SHMEM\_THREAD\_MULTIPLE} thread level, all \openshmem calls
+are thread-safe .i.e., any two concurrently running threads may make \openshmem calls.
```

Comment:
I think there's some whitespace issues. How about just:

are thread-safe. That is, any two concurrently running threads may make \openshmem calls.